02-wrangling.Rmd

```diff
@@ -256,9 +269,9 @@ lang_no_delimiter <- separate(lang_messy_longer,
 lang_no_delimiter
 ```
 
-You might notice in the table above the word `<chr>` appears beneath each of the column names. The word under the column name indicates the data type of each column. Here all of our variables are "character" data types. Recall, a character data type is a letter or a number. In the previous example, `most_at_home` and `most_at_work`  were `<dbl>` (double) (you can verify this by looking at the tables in the previous sections), which is a numeric data type. This change is due to the delimiter "/" when we read in this messy data set. R read the columns in as character types, and it stayed that way after we separated the columns. 
+You might notice in the table above the word `<chr>` appears beneath each of the column names. The word under the column name indicates the data type of each column. Here all of our variables are "character" data types. Recall, a character data type is a letter or a number surrounded by quotes. In the previous example, `most_at_home` and `most_at_work`  were `<dbl>` (double) (you can verify this by looking at the tables in the previous sections), which is a numeric data type. This change is due to the delimiter "/" when we read in this messy data set. R read the columns in as character types, and it stayed that way after we separated the columns. 
```

Comment:
I think the bit on `mutate` needs its own header. Otherwise it will be hard to find when learners want to refer back to this.

Also, should we demonstrate some math with mutate? Or at least creating a new column? For example ratio of `most_at_work`/`most_at_home`?